src/main/scala/nl.knaw.dans.easy.dd2d/DepositProperties.scala

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (C) 2020 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.knaw.dans.easy.dd2d
+
+import better.files.File
+import org.apache.commons.configuration.PropertiesConfiguration
+
+import scala.util.{ Failure, Try }
+
+object DepositProperties {
+
+  def add(depositDir: File, state: String, description: String): Try[Unit] = Try {
+    val pathToPropertiesFile = (depositDir / "deposit.properties").toJava
+    val config = new PropertiesConfiguration() {
```

Comment:
`config` is never used. IntelliJ will warn you about this, but you have to listen to it.